Sources/ViewControllers/Settings/CreditsViewController.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  CreditsViewController.swift
+//  OpenFoodFacts
+//
+//  Created by Егор on 2/16/18.
+//  Copyright © 2018 Andrés Pizá Bückmann. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class CreditsViewController: UIViewController {
+
+    private var webView: UIWebView!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupWebView()
+    }
+
+    private func setupWebView() {
+        webView = UIWebView(frame: self.view.frame)
+        webView.backgroundColor = .white
+        webView.clipsToBounds = true
+        webView.delegate = self
+        webView.scrollView.indicatorStyle = .white
+        self.view.addSubview(webView)
+        guard let htmlCreditsFilePath = Bundle.main.path(forResource: "Credits", ofType: "html"),
```

Comment:
After adding the subview, add webView's constraints. WebView can be pinned to the superview, taking into account the safe area:

```
webView.snp.makeConstraints { (make) in
    if #available(iOS 11.0, *) {
        make.edges.equalTo(self.view.safeAreaLayoutGuide)
    } else {
        make.edges.equalToSuperview()
    }
}
```